test/server/routes/applications.test.js

```diff
@@ -34,6 +34,7 @@ describe('routes/applications.js', () => {
 
   it('should serve up the applications index page on the /manage-applications mount path', () => {
     const slug = '/manage-applications';
+    process.env.FUTURE_DISPLAY_FLAG = 'true';
```

Comment:
Add this parameter to test > env.js, then you won't have to keep importing it in your test

Reply:
I set it to null in some tests or not 'true' in order to test both sides of the logical branch.

Reply:
Oh I see, I should've been paying more attention to it!